### DISCUSSION TITLE
If everyone has a bot, bots don't become helpful

### DISCUSSION THREAD

**Discussion by jaipalsilla** [Alkaar/resy-booking-bot#114] [General]
This is one of the most popular Resy repo's on GitHub. Given the hundreds of bots in market today, as more people use a bot, its effectiveness starts declining. Do you all share my opinion? Curious to see what you all think. What can we do to increase the effectiveness of the bot?

**Comment by jaipalsilla** [comment]
@Alkaar I would love to chat with you about this if you have some time!

**Reply by Alkaar** [reply]
What exactly is it you want to chat about?

**Reply by jaipalsilla** [reply]
@Alkaar Thanks for getting back to me and appreciate your patience. Couple questions/thoughts below:
1. If everyone starts using a bot, the effectiveness of a bot significantly declines. This has happened with sneaker (or any retail) bots and even with high frequency trading. As soon as everyone uses it, its value decreases dramatically. If so, (i) how can we prevent this from happening and (ii) what can we do to have an edge vs. the many bots out there?
3. Do you think there's a way for bots to coexist peacefully with reservation services (Resy, OpenTable)?
